tests/test_modeling_lxmert.py

```diff
@@ -0,0 +1,408 @@
+# coding=utf-8
+# Copyright 2018 LXMERT Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import unittest
+
+from transformers import is_torch_available
+from transformers.testing_utils import require_torch, slow, torch_device
+
+from .test_configuration_common import ConfigTester
+from .test_modeling_common import ModelTesterMixin, ids_tensor
+
+
+if is_torch_available():
+    import torch
+    from transformers import (
+        LxmertConfig,
+        LxmertModel,
+        LxmertForPretraining,
+    )
+    from transformers.modeling_lxmert import LXMERT_PRETRAINED_MODEL_ARCHIVE_LIST
+
+    #
+
+
+class LxmertModelTester:
```

Comment:
This test file is really great. It would still need to depend on the mixin to make sure we test everything.